README.md

```diff
@@ -81,6 +81,7 @@ Ansible config and a bunch of Docker containers.
 * [TimeMachine](https://github.com/awlx/samba-timemachine) - Samba-based mac backup server
 * [Traefik](https://traefik.io/) - Web proxy and SSL certificate manager
 * [Transmission](https://transmissionbt.com/) - BitTorrent client (with OpenVPN if you have a supported VPN provider)
+* [Deluge] (https://dev.deluge-torrent.org/) - A lightweight, Free Software, cross-platform BitTorrent client.
```

Comment:
Please can you add this in the correct position alphabetically, and fix the broken markdown URL?